docs/architecture/managers/sessions-manager.md

```diff
@@ -103,17 +104,21 @@ session_manager_addr
     })
     .wait(ctx);
 ```
+#### Anycast<T>
+The handling of this messages is basically just calling the method `get_random_anycast_session` from the
+[`Sessions`][sessions] library to obtain a random Session and send a `T` message.
```

Comment:
```suggestion
[`Sessions`][sessions] library to obtain a random `Session` and forward the `T` message to it.
```

Reply:
Shall we mention here what happens to the return value of the delegated call?

Reply:
Added rust code with the function where it happens